brig/cmd/brig/commands/build_list.go

```diff
@@ -16,11 +19,14 @@ import (
 
 const buildListUsage = `List all installed builds.
 
-Print a list of all of the current builds.
+Print a list of the current builds starting from latest (in creation time) to oldest. By default it will print all the builds, use --count to get a subset of them.
 `
 
+var buildListCount int
+
 func init() {
 	build.AddCommand(buildList)
+	buildList.Flags().IntVar(&buildListCount, "count", 0, "The maximum number of builds to return. 0 for all")
```

Comment:
Request: can we create this to support `-c` as a shorthand?  I think using `IntVarP` may do the trick...

Reply:
yup, IntVarP did the trick!